frontend/src/components/MiniCal/MiniCal.tsx

```diff
@@ -5,34 +5,20 @@ import 'react-datepicker/dist/react-datepicker.css';
 import './datepicker_override.css';
 import styles from './minical.module.css';
 import { useDate } from '../../context/date';
+import { holidaysList } from 'util/holidays';
 
-/**startDate is inclusive, endDate is exclusive */
-type Holiday = {
-  startDate: Date;
-  endDate: Date;
-  holidayName: string;
+const isWeekday = (date: Date) => {
+  return 0 < date.getDay() && date.getDay() < 6;
 };
 
-const holidays: Holiday[] = [
-  {
-    startDate: new Date('2024-2-24'),
-    endDate: new Date('2024-2-28'),
-    holidayName: 'Febuaray Break',
-  },
-  {
-    startDate: new Date('2024-3-30'),
-    endDate: new Date('2024-4-8'),
-    holidayName: 'Spring Break',
-  },
-];
-
 const isHoliday = (date: Date) => {
-  for (const holiday of holidays) {
-    if (holiday.startDate <= date && date < holiday.endDate) {
-      return true;
-    }
-  }
-  return false;
+  return holidaysList.some(
+    (holiday) => holiday.startDate <= date && date < holiday.endDate
```

Comment:
Hey Dwain! I checked out your branch and I like how Thanksgiving and Winter Break are greyed out on the calendar. However, I think the logic here might be off, because I can't see the other holidays. For one day holidays, try date <= holiday.endDate? Also, be careful about what dates you're actually comparing here, because it's possible it's comparing the dates down to the minute/second instead of the days themselves, so account for that as well.

Reply:
Thanks for the catch Selena.